samples/order/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+  </head>
+  <body>
+    <h1>Go CDK Order Sample</h1>
```

Comment:
WDYT about embracing the actual functionality a bit more? I.e., instead of "Place an Order", "Convert an Image"?

And maybe the frontend could have a link to the converted image once the "order" is complete.

@eliben

Reply:
Can I implement the link by calling Bucket.SignedURL with nil options? It says it's not supported on all providers, but does it work on the big 3? 

I'd rather not implement the link myself (by reading from the bucket and writing to the browser) if all the providers have a way to link to a blob. That would indicate something missing from our API.

Reply:
> Can I implement the link by calling Bucket.SignedURL with nil options?
> It says it's not supported on all providers, but does it work on the big 3?

Yes.

Although, for at least one provider (GCS) you need to provide some more info at construction time (e.g., use an `access_id` query parameter).  You can likely ignore that for now.

Reply:
Feel free to add a `TODO` for this as well, and do it in a separate PR.

Reply:
Added the TODO.